src/Files/Helpers/Win32Helpers.cs

```diff
@@ -45,7 +45,10 @@ public static async Task InvokeWin32ComponentsAsync(IEnumerable<string> applicat
                 }
 
                 await connection.SendMessageAsync(value);
+                return true;
```

Comment:
How about also getting the response, and if it's "Success", then return `true`?

Reply:
Currently the ftp does not return any response for this request, but it's a good idea to change that.

Reply:
Either assume the operation completed successfully if it doesn't report the status or like you've said, update the ftp to also report "Success" 👍

Reply:
Forgot SendMessageAsync() actually returns a result, changed function to return that.